ansible/roles/operator-pipeline/templates/openshift/tasks/merge-pr.yml

```diff
@@ -27,7 +27,7 @@ spec:
       image: registry.access.redhat.com/ubi8-minimal
       script: |
         #! /usr/bin/env bash
-        set -xe
+        set -x
```

Comment:
To not print curl with bot token

Reply:
Don't you mean to use `set -e`? Also, please put a critical warning comment on this line. We risk exposing the token through the pipeline logs which partners can view.